UnitTests/BraintreeShopperInsightsTests/BTShopperInsightsClient_Tests.swift

```diff
@@ -223,11 +223,16 @@ class BTShopperInsightsClient_Tests: XCTestCase {
         XCTAssertEqual(mockAPIClient.postedAnalyticsEvents.first, "shopper-insights:venmo-presented")
     }
     
-    func testSendVenmoSelectedEvent_sendsAnalytic() {
+    func testSendVenmoSelectedEvent() {
```

Comment:
I think we can revert this test name change

Reply:
Oops didn't mean to change that